subprojects/parseq/src/test/java/com/linkedin/parseq/ParSeqUnitTestHelperTest.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.parseq;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertThrows;
+
+public class ParSeqUnitTestHelperTest {
```

Comment:
Another maybe a quite dumb question: I know that the change you made in `ParSeqUnitTestHelper` is to remove compile dependency on `testNg`. Even without this change, currently consumers using JUNit can still use `ParSeqUnitTestHelper` for their parseq task. So your change is mainly to avoid pulling testNg dependency transitively for consumers using JUnit, right?

Reply:
Yes. We need to stay compatible for existing customers, and offer clean classpath to new customer who use JUnit5+